core/dovecot/conf/ham.script

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
-tee >(rspamc -h {{ ANTISPAM_WEBUI_ADDRESS }} -P mailu learn_ham /dev/stdin) \
-	>(rspamc -h {{ ANTISPAM_WEBUI_ADDRESS }} -P mailu -f 11 fuzzy_del /dev/stdin) \
-	| rspamc -h {{ ANTISPAM_WEBUI_ADDRESS }} -P mailu -f 13 fuzzy_add /dev/stdin
+RSPAMD_HOST="$(getent hosts {{ ANTISPAM_WEBUI_ADDRESS }})"
```

Comment:
I this will not work. `getent hosts` returns multiple values:

```
bash-5.1# getent hosts antispam
fd09:d816:d504:beef::9  antispam  antispam
```
you need to use cut, sed or a bash array:
- `RSPAMD_HOSTS="$(getent hosts antispam | cut -d" " -f1)"`
- `RSPAMD_HOSTS="$(getent hosts antispam | sed 's/\s.*//')"`
- `RSPAMD_HOSTS=($(getent hosts antispam))`

Reply:
I don't know what I was thinking...